### PR TITLE
PIA-1832: Add first snapshot test for Welcome view

### PR DESCRIPTION
**Pull request by kp-said-rehouni**
**Summary**
This PR integrates snapshot testing for tvOS. It includes the first test for the Welcome screen and it adds the snapshot testing on CI

### CODE REVIEW

File: PIA VPN-tvOSTests/Snapshot/__Snapshots__/WelcomeViewSnapshotTest/testExample.1.png
Comment (kp-geneva-parayno) [inline PIA VPN-tvOSTests/Snapshot/__Snapshots__/WelcomeViewSnapshotTest/testExample.1.png:1]: hi said - question please, does this need to get pushed? thanks

File: PIA VPNTests snapshot/PIA_VPNTests_snapshot.swift
Comment (kp-geneva-parayno) [inline PIA VPNTests snapshot/PIA_VPNTests_snapshot.swift:1]: hi said - we can also exclude this, just for clean-up, right? thanks

File: PIA VPNTests snapshot/__Snapshots__/WelcomeViewSnapshotTest/test_Welcome.1.png
Comment (kp-geneva-parayno) [inline PIA VPNTests snapshot/__Snapshots__/WelcomeViewSnapshotTest/test_Welcome.1.png:1]: hi said - question please, does this need to get pushed? thanks

**Review by kp-geneva-parayno — COMMENTED** [review]
tried it locally. i had to witch `record: true` on the first run, then it passed on the succeeding runs.

File: PIA VPN-tvOSTests/Snapshot/__Snapshots__/WelcomeViewSnapshotTest/testExample.1.png
Comment (kp-said-rehouni) [reply PIA VPN-tvOSTests/Snapshot/__Snapshots__/WelcomeViewSnapshotTest/testExample.1.png:1]: This one can be removed, will update it. Thank you!

File: PIA VPNTests snapshot/PIA_VPNTests_snapshot.swift
Comment (kp-said-rehouni) [reply PIA VPNTests snapshot/PIA_VPNTests_snapshot.swift:1]: Yes, will update it. Thank you!

**Review by kp-geneva-parayno — APPROVED** [review]
Looks good, thanks for helping out on this, Said! Thanks!

**Pull request merged**